letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -230,13 +230,14 @@ def deploy_cert(self, domain, cert_path, key_path,
         if not vhost.enabled:
             self.enable_site(vhost)
 
-    def choose_vhost(self, target_name):
+    def choose_vhost(self, target_name, temp=False):
         """Chooses a virtual host based on the given domain name.
 
         If there is no clear virtual host to be selected, the user is prompted
         with all available choices.
 
         :param str target_name: domain name
+        :param bool temp: whether or not self.make_vhost_ssl shouldn't be called
```

Comment:
I think this can be improved. It doesn't reference caching and to me it's not clear if `make_vhost_ssl` is called if `temp` is `True` or `False`. Perhaps we should be explicit and say what happens if the value is `True`?